### ISSUE TITLE
Entries in search result are not loaded

### ISSUE THREAD

**Issue by hobovsky** [codewars/codewars.com#2279]
After visiting this search URL: https://www.codewars.com/kata/search/my-languages?q=&r[]=-4&r[]=-3&r[]=-2&r[]=-1&xids=completed&beta=false&order_by=total_completed%20desc

I can see that ~300 kata were found, but after scrolling to the bottom of the list, it's expanded only once or twice and then no more entries are loaded.

I thought that it is caused by multiple selection of ranks, but I think I was not able to reproduce it with any other combination of selected badges.

**To Reproduce**
Go to https://www.codewars.com/kata/search/my-languages?q=&r[]=-4&r[]=-3&r[]=-2&r[]=-1&xids=completed&beta=false&order_by=total_completed%20desc, note amount of `xxx Kata Found`, and scroll to the bottom of the list to load more entries.

**Expected behavior**
I would expect to be able to see all search results, loaded dynamically in chunks.

**Screenshots**

![image](https://user-images.githubusercontent.com/23709795/100528210-7b86de80-31da-11eb-9737-53311257742d.png)

![image](https://user-images.githubusercontent.com/23709795/100528276-4929b100-31db-11eb-9198-e8309a99da1b.png)

**Browser**
Checked on Opera and Chrome, without any extensions installed other than Tampermonkey for Opera, but I disable my user script.

**Additional context**
There are no errors in browser console. Other users confirmed the issue.

**Comment by FArekkusu** [comment]
Duplicate of #936
